server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts

```diff
@@ -1,9 +1,59 @@
 import { Router, Request, Response } from 'express';
 
+import { graphqlRequest } from '../../GraphqlHTTPRequest';
+import { CreateAddressRespone } from '../..//Models/ClinicalDetails/CreateAddress';
+import { CREATE_ADDRESS, CREATE_INVESTIGATION, ADD_BACKGROUND_DESEASES, ADD_SYMPTOMS } from '../../DBService/ClinicalDetails/Mutation';
+
 const clinicalDetailsRoute = Router();
 
 clinicalDetailsRoute.get('/', (request: Request, response: Response) => {
     response.send('Hello from Clinical Details route');
-})
+});
+
+clinicalDetailsRoute.post('/saveClinicalDetails', (request: Request, response: Response) => {
+    
+    const requestAddress = request.body.input;
+
+    const requestInvestigation = request.body.investigation;
+    
+    graphqlRequest(CREATE_ADDRESS, { input: {
+        ...requestAddress
+    }}).then((result: CreateAddressRespone) => {
+        graphqlRequest(CREATE_INVESTIGATION, { investigation: {
```

Comment:
cant you spread the requestInvestigation object and than add the isolationAddress to the new object